PicoMQTT/src/PicoMQTT/outgoing_packet.cpp:216:19: error: this statement may fall through [-Werror=implicit-fallthrough=]

I use you lib in combination with OpenDTU application running on an ESP32.
The project is written using PlatformIO.

The project defined the following `build_flags`:
```
build_flags =
    -DCOMPONENT_EMBED_FILES=webapp_dist/index.html.gz:webapp_dist/zones.json.gz:webapp_dist/favicon.ico:webapp_dist/favicon.png:webapp_dist/js/app.js.gz
    -DPIOENV=\"$PIOENV\"
    -Wall -Wextra -Werror
    -std=c++17
    -std=gnu++17
; -DPICOMQTT_DEBUG_TRACE_FUNCTIONS
build_unflags =
    -std=gnu++11
```

Compilation of the project fails with:
```
.pio/libdeps/generic_esp32/PicoMQTT/src/PicoMQTT/outgoing_packet.cpp: In member function 'virtual bool PicoMQTT::OutgoingPacket::send()':
.pio/libdeps/generic_esp32/PicoMQTT/src/PicoMQTT/outgoing_packet.cpp:216:19: error: this statement may fall through [-Werror=implicit-fallthrough=]
             state = State::sent;
             ~~~~~~^~~~~~~
.pio/libdeps/generic_esp32/PicoMQTT/src/PicoMQTT/outgoing_packet.cpp:217:9: note: here
         case State::sent:
         ^~~~
Compiling .pio\build\generic_esp32\libad3\Hoymiles\HoymilesRadio.cpp.o
Compiling .pio\build\generic_esp32\libad3\Hoymiles\HoymilesRadio_CMT.cpp.o
Archiving .pio\build\generic_esp32\lib2d3\libU8g2.a
cc1plus.exe: all warnings being treated as errors
*** [.pio\build\generic_esp32\lib89b\PicoMQTT\PicoMQTT\outgoing_packet.cpp.o] Error 1
```
Removing the `-Werror` flag work-around the problem.

Do you see a possibility to fix this?

--- a/src/PicoMQTT/outgoing_packet.cpp
+++ b/src/PicoMQTT/outgoing_packet.cpp
@@ -214,6 +214,7 @@ bool OutgoingPacket::send() {
         case State::ok:
             // print.flush();
             state = State::sent;
+            __attribute__ ((fallthrough));
         case State::sent:
             return true;
         default:

